tests/codes/eurocode/nen_en_1993_1_9_c2_2012/annex_a_determination_of_fatigue_load_parameters_and_verification_formats/test_formula_a_2.py

```diff
@@ -0,0 +1,65 @@
+"""Testing formula A.1 from NEN-EN 1993-1-9+C2:2012: Annex A - Determination of fatigue load parameters and verification formats."""
+
+import pytest
+
+from blueprints.codes.eurocode.nen_en_1993_1_9_c2_2012.annex_a_determination_of_fatigue_load_parameters_and_verification_formats.formula_a_2 import (
+    FormADot2CriteriaBasedOnDamageAccumulation,
+)
+from blueprints.validations import NegativeValueError
+
+
+class TestFormADot2CriteriaBasedOnDamageAccumulation:
+    """Validation for formula A.2 from NEN-EN 1993-1-9+C2:2012."""
+
+    def test_evaluation_if_ok(self) -> None:
+        """Test the evaluation of the result."""
+        # example values
+        d_d = 0.8  # -
+
+        form_a_1 = FormADot2CriteriaBasedOnDamageAccumulation(d_d=d_d)
+        # manually calculated result
+        manually_calculated_result = True
```

Comment:
This changes when changing from boolean to other format.

Reply:
Not relevant after comment above.